<ui version="4.0" >
 <class>toToolSettingUI</class>
 <widget class="QWidget" name="toToolSettingUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <widget class="toTreeWidget" name="Enabled" >
     <property name="selectionMode" >
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>Tool</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QComboBox" name="DefaultTool" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="TextLabel2" >
     <property name="toolTip" >
      <string>Select the default tool to pop up when opening a new connection.</string>
     </property>
     <property name="text" >
      <string>&amp;Default tool</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>DefaultTool</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="TextLabel1" >
     <property name="toolTip" >
      <string>Select which tools should show up in the menu and toolbar.</string>
     </property>
     <property name="text" >
      <string>&amp;Enabled tools</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>Enabled</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="TextLabel3" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>You must restart for any of these changes to have effect.</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>toTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>totreewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>Enabled</tabstop>
  <tabstop>DefaultTool</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Enabled</sender>
   <signal>selectionChanged()</signal>
   <receiver>toToolSettingUI</receiver>
   <slot>changeEnable()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
